### ISSUE TITLE
loss_box_reg graph is increasing

### ISSUE THREAD

**Issue by YradenRavid** [pytorch/vision#1256]
Hi,

I'm training faster rcnn with torchvision and loss_box_reg graph from loss_dict['loss_box_reg'] is slowly increasing. Everything else seems normal. Why is that?
https://github.com/pytorch/vision/blob/v0.3.0/references/detection/engine.py#L30

![Screenshot 2019-08-21_19-03-07-896](https://user-images.githubusercontent.com/39432202/63448365-6dc6a300-c446-11e9-8f21-528602517a06.png)

Thanks,

Yarden

**Comment by fmassa** [comment]
This is normal, and it is because the box regression loss is only applied to the positive boxes.
Given that the amount of positive boxes is very small in the beginning of training (due to the objectness classifier not being very good), the box regression loss is smaller. But as training goes on, we have better objectness, and thus more boxes to be regressed, and the regression loss increases a bit.

**Comment by zhanwenchen** [comment]
@fmassa A related question: is loss_box_reg also supposed to be bad (in my case, increasing) at first?
